src/main/webapp/i18n/de/exerciseAssessmentDashboard.json

```diff
@@ -10,6 +10,20 @@
             "pageHeader": "Bewertungs-Dashboard für die Übung",
             "testRunPageHeader": "Testlaufübungs-Dashboard für die Übung:",
             "yourStatus": "Dein Status",
+            "tutorParticipationStatus": {
+                "title": {
+                    "NOT_PARTICIPATED": "Nicht teilgenommen",
+                    "REVIEWED_INSTRUCTIONS": "Anweisungen geprüft",
+                    "TRAINED": "An Beispielabgaben trainiert",
+                    "COMPLETED": "Studenten Abgaben bewertet"
```

Comment:
it should be: "Abgaben der Studierenden bewertet"

Reply:
done @lschlesinger